snowplow-android-core/src/main/java/com/snowplowanalytics/snowplow/tracker/Subject.java

```diff
@@ -246,11 +259,11 @@ private void setLocation(Context context) {
      *
      * @param context the android context
      */
-    private void setCarrier(Context context) {
-        String carrier = getCarrier(context);
-        if (carrier == null)
-            return;
-        putToMobile(Parameters.CARRIER, getCarrier(context));
+    public void setCarrier(Context context) {
+        String carrier = Util.getCarrier(context);
+        if (carrier != null) {
+            putToMobile(Parameters.CARRIER, carrier);
```

Comment:
Can we rename the `putToMobile` thing? I know it's a hangover from earlier versions but still sounds a bit odd...